platformcomponents/desktop/toggle-button-overlay.json

```diff
@@ -0,0 +1,50 @@
+{
+  "toggleButtonOverlay": {
+    "figma": "https://www.figma.com/file/vdL18BATeJAIq2JvGAjRPD/Components---Web?node-id=48160%3A80022",
```

Comment:
Please change it to https://www.figma.com/file/NaNrfXjygZtRgMfHAFHjsp/Components---Windows?node-id=44859%3A103949&t=p8zl3QPbotGIIFT3-1
to avoid having Archived pages links